server/test/read-time.spec.js

```diff
@@ -0,0 +1,14 @@
+import { expect } from 'chai';
+import { readTime } from '../utils';
+
+
+describe('Read time checks', () => {
+  it('should successfully return read time of a content', async () => {
+    const readtime = await readTime('<span>Like seriously</span> <p>the car and the bus</p>');
```

Comment:
Good test cases, your variable naming should follow convention, or what do you think about this line?

Reply:
Ok, I will implement this.

Reply:
Done.